qiskit/transpiler/target.py

```diff
@@ -711,6 +792,8 @@ def _build_coupling_graph(self):
         self._coupling_graph.add_nodes_from([{} for _ in range(self.num_qubits)])
         for gate, qarg_map in self._gate_map.items():
             for qarg, properties in qarg_map.items():
+                if qarg is None:
+                    continue
```

Comment:
It might be worth adding a release note about the changes in this function, since I think there is a user-facing API change here?

Reply:
Added in: https://github.com/Qiskit/qiskit-terra/pull/8852/commits/f06d1b89ec38c0762df6b9352d1e21c3f1a2fad4